app/src/main/res/values/strings.xml

```diff
@@ -572,4 +572,17 @@
   <string name="administrator_controls_fragment_test_activity_label">Administrator Controls Fragment Test Activity</string>
   <string name="select_a_topic_to_start">Select a Topic to Start</string>
   <string name="revision_navigation_cards_header">Continue Studying</string>
+  <string name="info_tab_content_description">Info tab</string>
+  <string name="revision_tab_content_description">After completing a lesson, revise your concepts in the revision tab.</string>
+  <string name="practice_tab_content_description">Practice tab</string>
+  <string name="lessons_tab_content_description">Start learning here in the lessons tab.</string>
```

Comment:
I feel like it would help me to see a video showing these in context. @JishnuGoyal is it possible for you to provide one? I can't really contextualize it otherwise.

If this is for the spotlight then I think the existing text seems fine; "Play a lesson" is a bit terse.

Also is it possible to fix the "Practice tab" description (again if it's for the spotlight)? Something like "Practice your skills here after learning them in the lessons." would do, I think.